dsew_community_profile/delphi_dsew_community_profile/constants.py

```diff
@@ -59,7 +59,12 @@ class Transform:
     "confirmed covid-19 admissions": {
         "is_rate" : False,
         "api_name": "confirmed_admissions_covid_1d_7dav"
+    },
+    "booster": {
+        "is_rate" : False,
+        "api_name": "booster_7dav"
```

Comment:
I changed it to these, let me know what you think:

"fully vaccinated": {
        "is_rate" : False,
        "api_name": "`covid_fully_vacc_num_7dav`"
    },
    "booster dose since": {
        "is_rate" : False,
        "api_name": "`covid_booster_num_7dav`"
    },
    "booster doses administered": {
        "is_rate" : False,
        "api_name": "`covid_total_booster_7dav`"
    },
    "doses administered": {
        "is_rate" : False,
        "api_name": "`covid_total_doses_7dav`"
    }

Reply:
Looks pretty good! There might be some confusion of num vs total ([CPR naming thread here](https://delphi-org.slack.com/archives/C013AH5N01E/p1643045140069100)), but I think this is fine until we need final approval for the names.

Reply:
Hmm for some reason I can't access that specific link even after logging in (it must be some of my Slack settings). I can always change it like you said after the final approval. 

Thanks for your review, Nat!